### PR TITLE
update rosjava_extras

### PR DESCRIPTION
**Pull request by talregev**
update to gradle 4.10.2
update cv_bridge_javacv to opencv 3.4.3 and support also android arm64.

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
```
Comment (jubeira) [inline gradle/wrapper/gradle-wrapper.properties:5]: Please revert this change; we were already using Gradle `4.10.2`.

**Review by jubeira — APPROVED** [review]
Thanks @talregev!
I've left a minor comment; otherwise LGTM! Please rebase the code to a single commit if possible.

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (talregev) [reply gradle/wrapper/gradle-wrapper.properties:5]: I just move the line. it catch the eye when the last line is the longest. more easy to catch the version. 
May  I keep it here?

Comment (jubeira) [reply gradle/wrapper/gradle-wrapper.properties:5]: fair enough :+1:
